src/components/Facets.tsx

```diff
@@ -0,0 +1,80 @@
+import { Facets as FacetsCompoundComponent } from './Filters/Facets';
```

Comment:
I didn't catch this for Nidhi's PR but could we rename the compound components for Facets and Filters to something else? I'd rather not have confusion when talking about components in the future. Maybe `FacetsProvider` since all it does is provide some behavior and Context setup? I'd avoid something like `CompoundFacets` or `FacetsCompoundComponent` because those names are imo more historical than descriptive

Reply:
I see. Sounds good to me! Will update